src/app/version-handler.ts

```diff
@@ -0,0 +1,105 @@
+import { get } from 'https';
```

Comment:
I think we can use the `net` module which has better support for proxy

Reply:
You mean this module `https://nodejs.org/docs/latest-v10.x/api/net.html#net_net`?

Reply:
No, [this](https://github.com/electron/electron/blob/master/docs/api/net.md) module